.travis.yml

```diff
@@ -1,13 +1,18 @@
 sudo: required
 language: java
 dist: trusty
+git:
+  # pull everything for safe commits comparison before deploying
+  depth: false
 services:
 - docker
 env:
   global:
   - DOCKER_VERSION=1.11.1-0~trusty
   - DOCKER_COMPOSE_VERSION=1.7.0
   - COMMIT=${TRAVIS_COMMIT::8}
+  # GH_TOKEN
+  - secure: "U5MNSISi7mUv+oDT9wwnrv2i90CaD6mSHrkkzW9tChikMsssq8cnWO8KwQ226Qbv2V16dLHr8sq504At2dKOzsxbBxLOkH7OXEt1uCrMErrMVL82XmKkThD6KTWSiDRdYLFoNbGv/q0B+U6ZZwPdTDibVquBw71cNDUuq8GdivCyT9f/7GQNUTVHjHmFRlU3lTAx9Z5rSKW6ZyL4tVFheTNNYO758Swg2pVVXRQbO9ZOfW7Mg5OKuR1i1gtpB4gpTtcVPykE3hEPd/loOHUZSQZu91c8uSk3Oxp1Pk8vKsUseebQoyaGXxnL1LZtNpbZHz+6p9PzlAEWIWNmUme0ji1B42zXgMxY2QNNj4Y27aNj9eiOqI6oUBjN1qyErgH467DEzMsbMs/weYW9KlwiJ9OfIHvRZ+BM5uxyeBs/xSb5UBlULmecsTDI5xWYJmv70iOLmSNZ4eRjFUkge89sGi/EgJEbLKE4Z8BXRbmdIMVycsZTfirS0k85NkWgNIQ4dMBhm5V28kZaPiO/kri4FXoge/Zbvqt1FkTY/puEh+AC7cP9S+PjpdTe9vtHc78BXP0cIrS/JL/wELiQumSDqXSnnQ+1OCB8+LLAMIr2tfA5YXC44UoyyaoMSWk7RO6KyUVgYDk3tTU7SFwsyeYVBGXr8X21tMxHE0Yu4nbS/FM="
```

Comment:
@yurishkuro I have added just `GH_TOKEN`, we do the same in OT contrib in many repos.

I have generated the token myself, we could switch to bot, but I am not sure if we have one.